mobile.js

```diff
@@ -0,0 +1,27 @@
+const eventMobile = document.querySelector('.ic_menu');
+
+const target1 = document.querySelector('.fa-sharp');
+const target2 = document.querySelector('.fa-xmark');
+
+const modal = document.querySelector('.menu-mobile');
+
+eventMobile.addEventListener('click', () => {
+  target1.classList.toggle('hide');
+  target2.classList.toggle('show');
+  modal.classList.toggle('hide');
+});
+
+document.documentElement.addEventListener('click', (e) => {
+  if (e.target.matches('#link1')) {
+    modal.classList.toggle('hide');
+  }
+  if (e.target.matches('#link2')) {
+    modal.classList.toggle('hide');
+  }
+  if (e.target.matches('#link3')) {
+    modal.classList.toggle('hide');
+  }
+  if (e.target.matches('#link4')) {
+    modal.classList.toggle('hide');
+  }
```

Comment:
Well done implementing this milestone. 

I see a couple of things that can help make your code better.

- Your Javascript variable naming does not follow best practices. A professional developer will write the code for the future self and for the "other guy" not just for the machine. Any code you write is never written just once rather it will sit down and wait for the future guy and make him miserable. Hopefully, that future guy won't be you. 
**_You should name your variables in a way that reveals the intention behind it. This way they become searchable and easier to understand after a person sees it._**

For example 👇🏾 

```javascript
// Bad variable name
const modal = document.querySelector('.menu-mobile'); //Modal is a general name and not specific

// Good variable name
const menuIcon = document.querySelector('.menu-mobile'); // Or anything that easily tell what it is.
```

```javascript
// Bad variable names
const eventMobile = document.querySelector('.ic_menu'); // What does this mean?
const target1 = document.querySelector('.fa-sharp');
const target2 = document.querySelector('.fa-xmark');

// Good variable names
const target1 = document.querySelector('.fa-sharp'); // I don't even know what this variable is. 
const closeIcon = document.querySelector('.fa-xmark');

```

#### Kindly consider renaming your variable names to reflect what they are. Please read [this article](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices) to learn more about writing clean javascript code.

Reply:
@bobb-Rob Thank you for the suggestions! I've worked on them 👍